support-frontend/assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYouBlurb.jsx

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+// ----- Render ----- //
+
+function ContributionThankYouBlurb() {
+  return (
+    <div className="gu-content__blurb gu-content__blurb--thank-you">
```

Comment:
this component and the component that appears in the initial page could likely be the same component but take an appearance prop to define if they go over white or over yellow

Reply:
Added to TBD